<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSelector</class>
 <widget class="QWidget" name="SystemSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="createFileSystem_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать файловую систему</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseFileSystem_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать файловую систему</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="fileSystemName_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
